ui/src/app/components/demographic-survey-v2.tsx

```diff
@@ -0,0 +1,794 @@
+            >
+              11. Do you have a physical, cognitive, and/or emotional condition
+              that substantially limits one or more life activities not
+              specified through the above questions, and want to share more?
+              Please describe.
+            </div>
+            <TextInput
+              onChange={(value) => onUpdate('disabilityOtherText', value)}
+              value={survey.disabilityOtherText || ''}
+              style={{ width: '50%' }}
+            />
+          </FlexColumn>
+          <SemiBoldHeader style={{ marginBottom: '0.5rem' }}>
+            Other Questions
+          </SemiBoldHeader>
+          <Divider style={{ marginTop: '.25rem' }} />
+          <div style={styles.question}>12. Year of birth</div>
+          <FlexRow style={{ alignItems: 'center', marginBottom: '1rem' }}>
+            <NumberInput
+              onChange={(value) => onUpdate('yearOfBirth', value)}
+              disabled={survey.yearOfBirthPreferNot}
+              min={minYear}
+              max={maxYear}
+              value={survey.yearOfBirth || ''}
+              style={{ width: '4rem' }}
+            />
+            <CheckBox
+              checked={survey.yearOfBirthPreferNot}
+              onChange={(value) => {
+                onUpdate('yearOfBirth', null);
+                onUpdate('yearOfBirthPreferNot', value);
+              }}
+              style={{ marginLeft: '1rem' }}
+            />
+            <label
+              htmlFor='show-again-checkbox'
+              style={{
+                ...styles.answer,
+                marginLeft: '8px',
+                marginRight: 'auto',
+              }}
+            >
+              Prefer not to answer
+            </label>
+          </FlexRow>
+          <MultipleChoiceQuestion
+            question={'13. Highest Level of Education'}
+            options={[
+              { label: 'No Education', value: EducationV2.NOEDUCATION },
+              { label: 'Grades 1-12', value: EducationV2.GRADES112 },
+              { label: 'College Graduate', value: EducationV2.COLLEGEGRADUATE },
+              { label: 'Undergraduate', value: EducationV2.UNDERGRADUATE },
+              { label: "Master's", value: EducationV2.MASTER },
+              { label: 'Doctorate', value: EducationV2.DOCTORATE },
+              {
+                label: 'Prefer not to answer',
+                value: EducationV2.PREFERNOTTOANSWER,
+              },
+            ]}
+            selected={survey.education}
+            onChange={(value) => onUpdate('education', value)}
+            style={{ marginBottom: '1rem' }}
+          />
+
+          <YesNoOptionalQuestion
+            question={disadvantagedBackgroundQuestion}
+            selected={survey.disadvantaged}
+            onChange={(value) => onUpdate('disadvantaged', value)}
+          />
+        </FlexColumn>
+      </FlexColumn>
+    );
+  }
+);
+
+export default DemographicSurvey;
```

Comment:
we have a few of these remaining, but our preference is now `export const` as part of the definition